content/blog/2021-10-30-november-21-community-gems.md

```diff
@@ -0,0 +1,197 @@
+```
+
+We're working on adding granularity support for this command and should have a
+release for this in the next few months.
+
+### [Is there a way to look at all of the experiments I've run and see the metrics and parameters associated with them?](https://discord.com/channels/485586884165107732/563406153334128681/900451895666155520)
+
+Thanks for asking @GuyAR! This is a common question that comes up.
+
+You can see all of your experiments and the associated metrics and parameters in
+a table in the terminal by running the following command:
+
+```dvc
+$ dvc exp show
+```
+
+This will give you a table that looks similar to this with all of this
+information.
+
+```dvctable
+┏━━━━━━━━━━━━━━━━━━━━━━━━━┳━━━━━━┳━━━━━━━━━┳━━━━━━━━━┳━━━━━━━━━┳━━━━━━━━━━┳━━━━━━━┳━━━━━━━━━━┓
+┃ neutral:**Experiment**              ┃ metric:**step** ┃     metric:**acc** ┃ metric:**val_acc** ┃    metric:**loss** ┃ metric:**val_loss** ┃ param:**lr**    ┃ param:**momentum** ┃
+┡━━━━━━━━━━━━━━━━━━━━━━━━━╇━━━━━━╇━━━━━━━━━╇━━━━━━━━━╇━━━━━━━━━╇━━━━━━━━━━╇━━━━━━━╇━━━━━━━━━━┩
+│ **workspace**               │    **3** │ **0.91389** │    **0.87** │ **0.20506** │  **0.66306** │ **0.001** │ **0.09**     │
+│ **data-change**             │    **-** │       **-** │       **-** │       **-** │        **-** │ **0.001** │ **0.09**     │
+│ │ ╓ 9405575 [exp-54e8a] │    3 │ 0.91389 │    0.87 │ 0.20506 │  0.66306 │ 0.001 │ 0.09     │
+│ │ ╟ 856d80f             │    2 │ 0.90215 │ 0.87333 │ 0.27204 │  0.61631 │ 0.001 │ 0.09     │
+│ │ ╟ 23dc98f             │    1 │ 0.87671 │    0.86 │ 0.35964 │  0.61713 │ 0.001 │ 0.09     │
+│ ├─╨ 99a3c34             │    0 │ 0.71429 │    0.82 │ 0.67674 │  0.62798 │ 0.001 │ 0.09     │
+│ │ ╓ 3b3a2a2 [exp-23593] │    3 │ 0.86885 │    0.46 │ 0.31573 │   3.7067 │ 0.001 │ 0.09     │
+│ │ ╟ 93d015d             │    2 │ 0.83197 │ 0.41333 │ 0.36851 │   3.4259 │ 0.001 │ 0.09     │
+│ │ ╟ d474c42             │    1 │ 0.79918 │ 0.43333 │ 0.46612 │    3.286 │ 0.001 │ 0.09     │
+│ ├─╨ 1582b4b             │    0 │ 0.52869 │    0.39 │ 0.94102 │   2.5967 │ 0.001 │ 0.09     │
+└─────────────────────────┴──────┴─────────┴─────────┴─────────┴──────────┴───────┴──────────┘
+```
+
+### [What's the recommended way to remove data that has been imported using `dvc import`?](https://discord.com/channels/485586884165107732/485596304961962003/898462029650735134)
+
+Great question @MadsO!
+
+This works the exact same as when you've added data with `dvc add`. So to remove
+data, you would run this command:
+
+```dvc
+$ dvc remove
+```
+
+### [How can I associate my Tensorboard experiment results, metrics, configs, etc. with DVC versioned datasets?](https://discord.com/channels/485586884165107732/563406153334128681/901272438468407326)
+
+There are a few approaches you can take to work with DVC datasets and
+Tensorboard.
+
+If you want to keep running your experiments with Tensorboard, you can use
+`dvc import` to associate the experiment results and metrics with your datasets.
+
```

Comment:
TBH, I'm pretty confused by this question, and I'm not sure I understand what was asked or how the answers are applicable. How does `dvc import` associate the experiment results and metrics with the datasets? It's pretty hard to tell from the user's Q&A, so I'm not really sure what to say here.

Reply:
🤔 Would it make more sense if we just highlighted that DVCLive here and removed the `dvc import` reference?

Reply:
Yes, I think it makes more sense, but I'm still not sure whether it actually answers the question because without more context I'm not really sure what the question was. Could we consider a different question or dropping this one?

Reply:
I remember the question from support dutty:

> We have a dataset_gen repository dedicated to generating/versioning large JSON datasets.  We recently began using DVC.  Now, we have multiple model training repositories that uses datasets generated/versioned by said dataset_gen repo.  Each model training repo uses Tensorboard for saving Experiment results/metrics/hyperparameters etc.  How should I associate an Experiment with a DVC versioned dataset?  I can think of logging the dataset_gen git commit hash to tensorboard but I feel there must be a better way.  Please let me know if you have a better way!

Before I got the chance to ask for more info user answered to himself:

> Solution found: dvc import

And I just added a little promotion of dvc exp and DVCLive.

I think that in this case the question was formulated strangely. 

My understanding of the user setup is that he has 2 separate repos, one for the dataset and others for experimentation. I think that he just found that using `dvc import` would generate a `.dvc` file on his experimentation repo so he can just log that to TensorBoard or track by git in that repo.

I'm not sure how they were importing the dataset before using `dvc import` as he didn't share more details for the setup.

Reply:
Ah, I see. I'll just remove this question and replace it with a different one. Thanks y'all!

Reply:
Thanks, @flippedcoder!